### PR TITLE
Add/update Saami and Romani languages

### PR DESCRIPTION
**Pull request by siebrand**
Requested at https://phabricator.wikimedia.org/T223524

### CODE REVIEW

File: data/langdb.yaml [hunk 1]
```diff
@@ -463,9 +463,19 @@ languages:
...
-  rmy: [Latn, [EU], Romani]
+  rmg: [Latn, [EU], romani rakripa]
+  # Waiting for a confirmed autonym
+  #rml: [Latn, [EU], Baltic Romani]
+  # Waiting for a confirmed autonym
+  #rmn: [Latn, [EU], Balkan Romani]
+  # Waiting for a confirmed autonym
+  #rmo: [Latn, [EU], Sinte Romani]
+  rmw: [Latn, [EU], kååle]
+  rmy: [Latn, [EU], řomani čhib]
```
Comment (amire80) [inline data/langdb.yaml]: As I noted in https://gerrit.wikimedia.org/r/#/c/mediawiki/core/+/511054/ , there should be a better reference for this name as well.

Comment (siebrand) [reply data/langdb.yaml]: How about this? https://translate.google.com/translate?sl=auto&tl=en&u=https%3A%2F%2Fwww.databazeknih.cz%2Fknihy%2Fromani-chib-ucebnice-slovenske-romstiny-275698

Comment (siebrand) [reply data/langdb.yaml]: P.s. If you only want sourced autonyms here, we should create a proper structure for sourcing in the language data, and every autonym should be sourced.

Comment (amire80) [reply data/langdb.yaml]: Thanks for the link! The book at the link says "romaňi čhib", not "řomani". That's precisely why I ask for sources. I see that this book is from Slovakia variety, but Vlax is spoken there, so it's probably right. If the people behind these patches think that "romaňi čhib" is a good name, I won't object it. The Wikipedia article should be corrected, too.

About your second, general question: I do my best to verify every name when I add new languages or review patches, and I mention the sources in commit messages, review messages, or Phab tasks.

File: data/langdb.yaml [hunk 2]
```diff
@@ -507,10 +517,13 @@ languages:
   shi: [shi-latn]
   shn: [Mymr, [AS], လိၵ်ႈတႆး]
   si: [Sinh, [AS], සිංහල]
+  sia: [Cyrl, [EU], аккала]
```
Comment (amire80) [inline data/langdb.yaml]: I'm not sure it's right. The only autonym I can find is at http://elibrary.karelia.ru/page2010.php?book=2090&page=3 (you may have to create a free account to see it). And perhaps it should be in the Latin alphabet.

File: data/langdb.yaml [hunk 3]
```diff
@@ -463,9 +463,19 @@ languages:
...
   rmf: [Latn, [EU], kaalengo tšimb]
-  rmy: [Latn, [EU], Romani]
+  rmg: [Latn, [EU], romani rakripa]
+  # Waiting for a confirmed autonym
+  #rml: [Latn, [EU], Baltic Romani]
+  # Waiting for a confirmed autonym
+  #rmn: [Latn, [EU], Balkan Romani]
+  # Waiting for a confirmed autonym
+  #rmo: [Latn, [EU], Sinte Romani]
+  rmw: [Latn, [EU], kååle]
```
Comment (amire80) [inline data/langdb.yaml]: Is there another source for this spelling of the name other than Wikipedia?

File: data/langdb.yaml [hunk 4]
```diff
@@ -507,10 +517,13 @@ languages:
   shi: [shi-latn]
   shn: [Mymr, [AS], လိၵ်ႈတႆး]
   si: [Sinh, [AS], සිංහල]
+  sia: [Cyrl, [EU], аккала]
   simple: [en-simple]
   sjd: [Cyrl, [EU], Кӣллт са̄мь кӣлл]
   sje: [Latn, [EU], bidumsámegiella]
+  sjk: [Latn, [EU], kemi]
```
Comment (amire80) [inline data/langdb.yaml]: Should it be "kemi" or perhaps "kemi sámi"?

File: data/langdb.yaml [hunk 5]
```diff
@@ -507,10 +517,13 @@ languages:
...
   sjo: [Mong, [AS], ᠰᡞᠪᡝ ᡤᡞᠰᡠᠨ]
+  sjt: [Cyrl, [EU], тер]
```
Comment (amire80) [inline data/langdb.yaml]: I'm also not sure about this autonym. Even if there is no standard orthography, I'm fine with an educated guess, but it should still probably include the "Saami" part.

**Comment by siebrand** [comment]
Retracting patch.

**Pull request closed without merging**